web/server/www/scripts/codecheckerviewer/ListOfRuns.js

```diff
@@ -81,24 +81,7 @@ function (declare, dom, ItemFileWriteStore, topic, Dialog, Button,
 
   function runNameFormatter(args) {
     var runData = args.runData;
-
-    var label = '<span class="link">' + runData.name + '</span>';
-
-    var ctuEnabled = runData.runCmd.indexOf('--ctu') !== -1;
```

Comment:
@gyorb Unfortunately with these changes we will lost this feature. If we would like to keep this we need to extend the `getRunData` API function to return with these tags (ctu, stats).

Reply:
What do you think about extending the new api call?
 `def getCheckCommand(self, run_history_id, run_id, tags):`
It could parse up the tags (analysis arguments) and return them.